src/display/annotation_layer.js

```diff
@@ -2145,6 +2156,26 @@ class PopupElement {
     this.#container.style.zIndex =
       parseInt(this.#container.style.zIndex) - 1000;
   }
+
+  forceHide() {
+    this.#wasVisible = this.isVisible;
+    if (!this.#wasVisible) {
+      return;
+    }
+    this.#container.hidden = true;
```

Comment:
To prevent any possible future inconsistencies in internal state, should we perhaps call `this.#hide();` here instead to "properly" hide it?

Reply:
The "problem" in using `#hide` or `#show` is that they're checking the `pinned` state.
So here we just hide whatever the state of the popup is (pinned or not).
We could of course use them but I had the feeling it was "too much" complexity for such a thing.